File: app/src/main/java/com/ditclear/paonet/EmptyFragment.java
package com.ditclear.paonet;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;


public class EmptyFragment extends Fragment {


    public static EmptyFragment newInstance() {
        
        Bundle args = new Bundle();
        
        EmptyFragment fragment = new EmptyFragment();
        fragment.setArguments(args);
        return fragment;
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        return inflater.inflate(R.layout.empty_layout, null);
    }

    @Override
    public void onViewCreated(View view, Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

    }

}

File: app/src/main/java/com/ditclear/paonet/EmptyFragment.kt
package com.ditclear.paonet

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup


class EmptyFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater!!.inflate(R.layout.empty_layout, null)
    }

    override fun onViewCreated(view: View?, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

    }

    companion object {


        fun newInstance(): EmptyFragment {

            val args = Bundle()

            val fragment = EmptyFragment()
            fragment.arguments = args
            return fragment
        }
    }

}
